### DISCUSSION TITLE
Error prompt displayed when using LXQt for First time

### DISCUSSION THREAD

**Discussion by Saeid008** [lxqt/lxqt#2472] [Q&A]
Hi guys,

I have installed LXQt for my Ubuntu 20 and for the first time using it, I see this error message in this picture. 
<img width="960" alt="LXQt " src="https://github.com/lxqt/lxqt/assets/69323214/91c5efaf-df48-4a1a-b35e-e82ed8037655">
I have no idea how to fix that.
Anybody can help me?

**Comment by tsujan** [comment]
> I have installed LXQt for my Ubuntu 20...

The LXQt version in Ubuntu 20 should be a relic of old times: extremely old and outdated.

**Reply by stefonarch** [reply]
It looks like there is no window manager installed, there should be a list with at least one item. Try in recovery mode > root prompt `apt install openbox` or in tty the same with sudo.

And yes, 22.04 + backports is recommended to have the recent version.

**Reply by tsujan** [reply]
> It looks like there is no window manager installed

Yes, but the problem is that we don't remember bugs in very old versions.

**Reply by Saeid008** [reply]
it's strange that this list is empty.
The prompt makes us to select a file related to default windows manager. I think lxqt should put the file on GitHub so users manually download that file on the Ubuntu and select it. @tsujan

**Reply by tsujan** [reply]
LXQt is "agnostic" toward WMs. So, it doesn't/shouldn't provide any.

Providing a WM with an `lxqt` meta-package is a distro job. Apparently all distros do it nowadays, such that if the user installs the `lxqt` meta-package of his distro (Ubuntu in your case), he'll have one WM at least.
